lms/static/js/ccx/schedule.js

```diff
@@ -251,9 +265,25 @@ var edx = edx || {};
 	    }
 	},
 
+	valid_dates: function(start, due) {
+          var errorMessage;
+          if (_.isEmpty(start) && !_.isEmpty(due)) {
+            errorMessage = "Please enter valid start date and time";
```

Comment:
Here's where the error message is defined -- you should call `gettext()` here, instead of in `schedule.js`. Same with the other error message defined below.